scripts/manage-imports.py

```diff
@@ -26,18 +26,16 @@ def ol_import_request(item, retries=5, servername=None, require_marc=True):
     """Requests OL to import an item and retries on server errors.
     """
     # logger uses batch_id:id for item.data identifier if no item.ia_id
-    _id = item.ia_id or "%s:%s" % (item.batch_id, item.id)
-    logger.info("importing %s", _id)
+    logger.info("importing %s", item.ia_id)
```

Comment:
Now all IDs (for reporting purposes) are standardized as the field in the db, rather than using 2 different forms depending on the situation (which was throwing off the http://openlibrary.org/admin/imports dashboard)